qcodes/instrument/base.py

```diff
@@ -357,6 +356,35 @@ def name_parts(self) -> List[str]:
     @property
     def full_name(self) -> str:
         return "_".join(self.name_parts)
+
+    def _raise_if_abstract(self) -> None:
+        """
+        This method is run after the initialization of a subclass of
+        `InstrumentBase`. If we have a chain of base classes like so:
+        InstrumentBase -> Instrument -> VisaInstrument -> Driver
+        This method is run after the instantiation of the class `Driver`.
```

Comment:
Although the comment is accurate, I think calling out the chain of base-classes is a bit of a misdirect and makes this a bit confusing. I ended up thinking about whether we were hooking in between __init__ calls, which I think was how the first implementation of this feature worked?

Might be better to remove the chain of base classes and only give the first sentence.

Reply:
No this is copied from the old implementation, my comment above was meant as a reminder to update this